### DISCUSSION TITLE
配置PaoPaoDNS和PaoPaoGateway后会导致不以ipv6优先？

### DISCUSSION THREAD

**Discussion by biggbuddy** [kkkgo/PaoPaoDNS#197] [问答和帮助] [answered]
如题， Proxmox All In Boom环境 ， OpenWrt / PaoPaoDNS /PaoPaoGateway都放在PVE中，在配置 PaoPaoDNS /PaoPaoGateway之前测试发现网络是IPV6优先， OP中配上  PaoPaoDNS /PaoPaoGateway 后， 发现成了IPV4优先， 请教一下是什么原因？ 有办法不改变这个吗？

**Comment by kkkgo** [comment]
参考文档的IPV6选项。
另外如果测试域名经过了paopaogateway，那么测试结果取决于你的节点出口是否以IPV6优先。

**Reply by biggbuddy** [reply]
我是在 https://ipw.cn/  这个网站来测试是否以ipv6优先，也试了PaoPaoDNS中ipv6的每一个选项， 貌似都不是以ipv6优先。
另外查看了paopao gateway中Clash的连接日志， 用ipw搜索，看到虽然经过了paopao gateway， 但命中的规则是DomainSuffix(cn)， 走的链路是DIRECT

**Reply by kkkgo** [reply]
1、这个网站的测试服务器位于日本。按境外自动分流会到paopaogateway。https://www.itdog.cn/ping_ipv6/6.ipw.cn  
2、paopaogateway不存在任何规则，规则都是你自己塞进去的。

**Reply by biggbuddy** [reply]
我的意思是说，我在Clash的连接日志中查到的是走的是DIRECT， 这个应该不会经过机场的节点？

**Reply by biggbuddy** [reply]
IPV6我配的是yes

**Reply by kkkgo** [reply, marked as answer]
yes就是境外不提供IPv6，“这个网站的测试服务器位于日本。按境外自动分流会到paopaogateway”
你配置了规则让他走DIRECT那的确不会走节点，但paopaogateway本身并没有IPv6地址。

**Reply by biggbuddy** [reply]
明白。其实没啥用， 只是测试的时候碰到问一下， 不纠结。
感谢
